src/ghdcbot/engine/snapshots.py

```
@@ -253,21 +252,19 @@ def _collect_snapshot_data(
+    for req in pending_requests:
+        issue_requests_data.append({
+            "request_id": req.get("request_id"),
+            "discord_user_id": req.get("discord_user_id"),
+            "github_user": req.get("github_user"),
+            "owner": req.get("owner"),
+            "repo": req.get("repo"),
+            "issue_number": req.get("issue_number"),
+            "issue_url": req.get("issue_url"),
+            "created_at": req.get("created_at"),
+            "status": req.get("status"),
+        })
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)